vllm/v1/engine/async_llm.py

```diff
@@ -110,6 +113,15 @@ def from_engine_args(
     def _get_executor_cls(cls, engine_config: EngineConfig):
         return GPUExecutor
 
+    async def abort_request(self, request_id: Union[str,
+                                                    Iterable[str]]) -> None:
+        if isinstance(request_id, str):
+            request_id = [request_id]
+        self.engine_core.abort_requests_async(
```

Comment:
I think this should be `await self.engine_core.abort_request_async`

Reply:
Good catch. I expected the `examples/openai_completion_client.py` to catch any such errors - I'll add some unit tests.